build/scripts/build.ps1

```diff
@@ -179,6 +179,21 @@ function Build-Artifacts() {
 
     if ($buildAll) {
         Build-ExtraSignArtifacts
+    }
+
+    if ($pack) {
+        Build-NuGetPackages
+
+        if ($cibuild -or $official) { 
+            Build-DeployToSymStore
+        }
+    }
+
+    if ($sign) {
```

Comment:
Signing should be done before deploying to SymStore. It probably works now since DeployToSymStore uses hard-links to the files in the output directory that are going to be signed, but ideally we would not rely on that.

Reply:
Will reorder that.